go/vt/vtorc/collection/collection.go

```diff
@@ -260,8 +260,8 @@ func (c *Collection) removeBefore(t time.Time) error {
 	// get the interval we need.
 	if first == len(c.collection) {
 		c.collection = nil // remove all entries
-	} else if first != -1 {
-		c.collection = c.collection[first:]
+	} else {
+		c.collection = c.collection[first+1:]
```

Comment:
I think it should be `c.collection[first+1:]` instead of `c.collection[first:]`. 
It gives incorrect result for `c.collection[first:]`.
Please confirm.

Reply:
I looked at the code, and I think you're right.